src/state/mutations.js

```diff
@@ -17,6 +17,23 @@ export const CREATE_FILE = gql`
   }
 `;
 
+// Mutation to upload a new version of an existing file to the study-creator
+export const CREATE_FILE_VERSION = gql`
```

Comment:
Do you need both this and `CREATE_FILE`? The `fileId` is optional, so I would think you just need one.